### ISSUE TITLE
Mobile version

### ISSUE THREAD

**Issue by matthiasbock** [matthiasbock/BooleSim#16]
It would be cool to render a more compact interface on small screens (smartphones). Does of course depend on the smartphone's browser's html5/js capabilities...

**Comment by chaitan3** [comment]
Yes, it would be cool, but what be the use case? I'm sure most of the researchers would use biographer in a desktop browser. Users won't store networks on their mobile phones. And the website should display correctly on a tablet.

**Comment by TillS** [comment]
Let's postpone this. It really is not necessary and I don't really see a use for it either...

**Comment by matthiasbock** [comment]
Screen too small. Maybe we could only display icons on the buttons in such a case, without text ...

![screen_too_small](https://f.cloud.github.com/assets/1587578/1012600/0964465c-0b73-11e3-8956-b6361c44f7ac.jpg)
